intelmq/tests/bots/experts/uwhoisd/test_expert.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+import os
+import unittest
+import json
+
+import intelmq.lib.test as test
+from intelmq.bots.experts.uwhoisd.expert import UniversalWhoisExpertBot
+
+
+EXAMPLE_INPUT = {"__type": "Event",
+                 "source.fqdn": "www.cert.at",
+                 "time.observation": "2015-01-01T00:00:00+00:00"
+                 }
+EXAMPLE_OUTPUT = {"__type": "Event",
+                  "source.fqdn": "www.cert.at",
+                  "extra.whois": "Whois record",  # will be replaced in the code below
+                  "time.observation": "2015-01-01T00:00:00+00:00"
+                  }
+
+
+@test.skip_exotic()
+@unittest.skipIf(not os.environ.get('INTELMQ_UNIVERSALWHOIS_TEST'), "UniversalWhois test skipped, as we dont wanna overload public instance")
```

Comment:
Maybe I'm missing something, but doesn't this bot connect to localhost? (which won't work)

Reply:
We don't have a public instance of uWhoisd (yet?), so I ran it locally for testing and it worked when I did that:

```
INTELMQ_TEST_EXOTIC=1 INTELMQ_UNIVERSALWHOIS_TEST=1 nosetests-3.4 -v intelmq/tests/bots/experts/uwhoisd/test_expert.py
```

Reply:
Ah, ok. I'll check what options we have to test that bot.